### PR TITLE
CAL-105 remove unneeded blueprint config for mpegts udp stream monitor

### PR DESCRIPTION
**Pull request by glenhein**
#### What does this PR do?
Remove unneeded blueprint config for mpegts udp stream monitor. The functionality had been moved to the mpegts input transformer.
#### Who is reviewing it?
@jrnorth 
@bdeining 
@rzwiefel 
@lcrosenbu 
#### How should this be tested?
Build experimental and ingest a udp stream.
#### What are the relevant tickets?
CAL-105
#### Checklist:
- [ ] Documentation Updated
- [X] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/PacketBuffer.java [hunk 1]
```diff
@@ -85,6 +85,8 @@ public void accept(Visitor visitor) {
         }
     };
 
+    private static final long ACTIVIITY_LOG_PERIOD = TimeUnit.SECONDS.toMillis(10);
```
Comment (jrnorth) [inline catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/PacketBuffer.java]: Typo: `activiity`

Comment (glenhein) [inline catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/PacketBuffer.java]: fixed

**Comment by jlcsmith** [comment]
Built and tested.  LGTM 👍

**Comment by bdeining** [comment]
:+1:

**Comment by coyotesqrl** [comment]
👍

**Pull request merged**